tutorials/csvformatter/README.md

```diff
@@ -0,0 +1,64 @@
+##CsvFormatter Example
+
+`CsvFormatter` converts the incoming POJO to CSV(by default) or custom delimiter('|', ':' etc.) separated string.  
+
+Accordingly, we have additional config file for formatting preferences (`/src/main/resources/schema.json`) besides the common properties file (`/src/main/resources/META-INF/properties.xml`). 
+
+Users can choose the application and additional configuration file to use during launch time. In this example, we use the files mentioned above to customise the schema and configure the operator properties.
+
+
+####**Update Properties from properties.xml - This is needed to run the example:**
+
+- Update these common properties in the file `/src/main/resources/META-INF/properties.xml`:
+
+| Property Name  | Description |
+| -------------  | ----------- |
+| dt.application.CustomOutputFormatter.operator.jsonParser.prop.sleepTime |sleep time for the container |
+| dt.application.CustomOutputFormatter.operator.jsonParser.port.out.attr.TUPLE_CLASS | expected POJO object output of the JSONParser operator |
+| dt.application.CustomOutputFormatter.operator.HDFSOutputOperator.prop.filePath | output file path for the records after formatting |
+| dt.application.CustomOutputFormatter.operator.HDFSOutputOperator.prop.outFileName | output file name for the records to be written after formatting |
+
+
+####**Update Properties from Application.java - This is needed to customise Formatter:**
+
+```
+    CsvFormatter formatter = dag.addOperator("formatter", CsvFormatter.class);
+    formatter.setSchema(SchemaUtils.jarResourceFileToString(filename));
+    dag.setInputPortAttribute(formatter.in, PortContext.TUPLE_CLASS, PojoEvent.class);
+```
+
+'filename' above is the variable for storing schema file in the example.
+
+The input port attribute can be configured using properties.xml by setting the below property as well.
+
+```dt.application.CustomOutputFormatter.operator.formatter.port.input.attr.TUPLE_CLASS```
+
+
+
+
+####**Sample Run:**
```

Comment:
@shubham-pathak22   I can see that it is reflected in the UI. Please check it in Github UI.